examples/basic-example/requirements.txt

```diff
@@ -0,0 +1,2 @@
+mirascope==0.1.0
+openai==1.6.0
```

Comment:
I realized later that yeah mirascope will install this so not necessary unless we want to make it clear you're installing openai?

Reply:
i think this is fine to keep, we will update this as necessary with different models